purchase_order_supplier_phone/views/purchase_order_views.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="purchase_order_form_inherit" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
```

Comment:
We normally do not even add `inherit` as it doesn't mean much, although it does not do a harm if you add it.